src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -737,14 +737,14 @@ s32 func_80A98034(EnKo* this, PlayState* play) {
     s32 result;
 
     if (this->unk_1E8.unk_00 != 0) {
-        if ((this->skelAnime.animation == &gObjOsAnim_8F6C) == false) {
+        if ((this->skelAnime.animation == &gKokiriBoyBlockingAnim) == false) {
             Animation_ChangeByInfo(&this->skelAnime, sAnimationInfo, ENKO_ANIM_29);
```

Comment:
You probably have enough information to name the enum members as well now.

Reply:
Do you mean the `EnKoAnimation` enum? The other actors I've seen with documented animation names don't rename the members of those enums. ([Mido](https://github.com/zeldaret/oot/blob/master/src/overlays/actors/ovl_En_Md/z_en_md.c#L58), [Zora](https://github.com/zeldaret/oot/blob/master/src/overlays/actors/ovl_En_Zo/z_en_zo.c#L312))

Reply:
they only arent named cause no one has felt like doing it. You can rename them if you feel like, but its also something that can be done when more documentation of the code occurs

Reply:
Named the enum members